test/utils.js

```diff
@@ -183,6 +185,21 @@ export const separator = length => {
   console.log(`\n${separator}\n`);
 };
 
+export const makePublic = () => {
```

Comment:
Let's have a more explicit name for this one, like `makeUpdatePublic`.

Reply:
will change that ASAP